### DISCUSSION TITLE
mosdns不支持http的api配置

### DISCUSSION THREAD

**Discussion by Cause-jun** [kkkgo/PaoPaoDNS#148] [问答和帮助]
我注意到paopaodns中的mosdns相比较原来的mosdns少了api http的配置，请问未来有希望加上吗？虽然在博客中有说明千人企业用过，但还是想通过api暴露的指标使用Prometheus进行监控。

**Comment by kkkgo** [comment]
套一层AdGuard Home就可以了，自带面板，也有相关exporter https://grafana.com/grafana/dashboards/20799-adguard-home-exporter/
